diff --git a/better_exposed_filters.module b/better_exposed_filters.module
index 28276d6..b449754 100644
--- a/better_exposed_filters.module
+++ b/better_exposed_filters.module
@@ -75,7 +75,7 @@ function better_exposed_filters_views_api() {
  * Unpacks sort_by and sort_order from the sort_bef_combine element.
  */
 function bef_sort_combine_submit($form, &$form_state) {
-  $combine_param = $form_state['#combine_param'];
+  $combine_param = empty($form_state['#combine_param']) ? $form_state['complete form']['#info']['sort-sort_bef_combine']['value'] : $form_state['#combine_param'];
   if (empty($form_state['values'][$combine_param])) {
     $form_state['values']['sort_by'] = $form_state['values']['sort_order'] = '';
   }
